### PR TITLE
Update-Papercups-Greeting

### LINKED ISSUE #4553: Suggestion: Documentation Button on Toolbar
## Is your feature request related to a problem?

This is a suggestion borne out of some recent research I've done where our focus customers had either explicitly requested more documentation on certain topics (such as best practices for naming) or had experienced issues when on-boarding non-technical team members into PostHog. 

Some of the content these users would want already exists and some of it doesn't, but discoverability for what documentation we do have is limited in PH and fragments users across multiple channels. That's what the feature request is intended to solve.

## Describe the solution you'd like

I'd suggest adding an item on the toolbar titled 'Help', 'Tutorials', 'Documentation', or similar. On this page we can feature a greater variety and depth of educational content than we currently do, without requiring the user to exit PH.

Clicking this toolbar button would open (in a new window, potentially) the [Tutorials main page](https://posthog.com/docs/tutorials), which can be a content hub for future tutorials. On this main page we can suggest existing channels (Slack, Repo, etc), feature any just-added content and enable users to use the existing search feature to find tutorials they need.

I'd suggest leaving the Papercups integration in place as an additional channel, with the addition of an autoresponse for any first contact, e.g: 'Thanks! We'll get back to you shortly. If you have a product question, don't forget to check our [documentation](https://posthog.com/docs/tutorials) too!'

## Describe alternatives you've considered

Other options could include adding a link to the bottom of each PH page (minimal discoverability) or adding to the 'My Account' drop-down in the top-right. 

Alternatively we could keep this content within the 'Home' tab and instead change the messaging, but this still feels hard to find. When I was looking for a documentation link recently, Home was the last place I looked. 

## Additional context

The existing solution is that the in-PH documentation right now is under the 'Home' tab, located at the bottom of the page, inside two drop-down menus. Other than this, users would have to leave PH and navigate manually to another source.

One of these current menus directs to the tutorials handbook. However it only surfaces a limited number of issues (it doesn't mention the 1-minute tutorials, for example) and links directly to these issues, so users may not know there is additional help content in the handbook. 

The other menu includes links to the repo (not ideal for non-technical users) and the Slack (may be high barrier to entry). The other way users can request assistance is via the Papercups integration, but this requires someone to be available to respond.

### PR DESCRIPTION
**Pull request by joethreepwood**
Updated Papercups greeting to point to existing docs.

## Changes

Updated Papercups.tsx at line 20 to give a new greeting for issue 4553

## Checklist

- [ ] All querysets/queries filter by Organization, by Team, and by User
- [ ] Django backend tests
- [ ] Jest frontend tests
- [ ] Cypress end-to-end tests
- [ ] Migrations are safe to run at scale (e.g. PostHog Cloud) – present proof if not obvious

### CODE REVIEW

File: frontend/src/lib/components/Papercups.tsx [hunk 1]
```diff
@@ -17,7 +17,7 @@ export function Papercups(): JSX.Element {
             subtitle="Ask us anything in the chat window below 😊"
             newMessagePlaceholder="Start typing…"
             primaryColor="#5375ff"
-            greeting="Hi! Send us a message and we'll respond as soon as we can."
+            greeting="We'll respond as soon as we can. You can also check posthog.com/docs if you need assistance."
```
Comment (paolodamico) [inline frontend/src/lib/components/Papercups.tsx:20]: Think maybe we want to keep the greeting? i.e. "Hi! We'll respond ...", also we could highlight the benefit of checking the docs which is getting an answer right away, so something like: "Hi! We'll respond as soon as we can. For an immediate answer you may want to search our docs (posthog.com/docs)"

**Review by paolodamico — APPROVED** [review]

File: frontend/src/lib/components/Papercups.tsx (hunk 1 above)
Comment (joethreepwood) [reply frontend/src/lib/components/Papercups.tsx]: Yep, made a quick tweak to it so that it's staying as short as possible.

Comment (joethreepwood) [reply frontend/src/lib/components/Papercups.tsx]: Made an additional update by adding in an 'awayMessage' field after discussion with Alex at PC

**Comment by paolodamico** [comment]
Adjusted the parameter name and the copy a tiny bit, will merge when tests pass

**Pull request merged**